src/main/java/core/basesyntax/FigureTwo.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class FigureTwo extends Figure {
+
+    private double firsSide; // длина
```

Comment:
There is a typo in the variable name `firsSide`. It should be `firstSide` to maintain clarity and consistency in naming conventions.